.github/workflows/netlifypreview.yml

```
@@ -0,0 +1,37 @@
+name: NetlifyPreview
+
+on:
+  pull_request_target:
+    types: [labeled, synchronize]
+
+jobs:
+  add-preview:
+    runs-on: ubuntu-20.04
```

Run on this year's Ubuntu.
```suggestion
    runs-on: ubuntu-22.04
```